### PR TITLE
Add Essent

### PR DESCRIPTION
**Pull request by TheLastProject**
**Description:**
Allows retrieving your meter information from Essent.

Note: I have not been able to test how it renders because I'm having issues with my Ruby installation.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#23513

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by TheLastProject** [comment]
Hmm, the page doesn't show up in the preview and I don't know why :(

File: source/_components/essent.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+comments: false
+sharing: true
+footer: true
+logo: essent.png
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 0.93
+ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/essent.markdown]: Please remove this -> `redirect_from`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
If all goes well you can now see the page in netlify

**Comment by TheLastProject** [comment]
Yup I can see it, and it looks fine to me. Please tell me if you want me to change anything :)

**Review by klaasnicolaas — APPROVED** [review]

**Comment by TheLastProject** [comment]
Parent has been merged :)

**Comment by klaasnicolaas** [comment]
Well then it's time to merge 😉 Always nice some new Dutch integrations 🎉

**Pull request merged**